spec/spec_helper.rb

```diff
@@ -14,6 +14,9 @@
 require "topological_inventory/core/ar_helper"
 TopologicalInventory::Core::ArHelper.load_environment!
 
+require 'manageiq/loggers'
+Rails.logger = ManageIQ::Loggers::Base.new(TopologicalInventory::Core::ArHelper.root.join("log/test.log"))
```

Comment:
Wouldn't it make more sense for it to be `TopologicalInventory::Core.root...`?

Reply:
Sure? I just left it where it was defined before. It was a private class method.